packages/techdocs-cli/package.json

```diff
@@ -50,16 +50,16 @@
     "ext": "ts"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.8.3",
+    "@backstage/backend-common": "^0.8.6",
     "@backstage/catalog-model": "^0.8.4",
-    "@backstage/techdocs-common": "^0.6.5",
+    "@backstage/techdocs-common": "^0.6.8",
     "@types/dockerode": "^3.2.1",
     "commander": "^6.1.0",
     "dockerode": "^3.2.1",
     "fs-extra": "^9.0.1",
     "http-proxy": "^1.18.1",
     "react-dev-utils": "^11.0.4",
     "serve-handler": "^6.1.3",
-    "winston": "3.2.1"
+    "winston": "^3.2.1"
```

Comment:
This fixes typing issues since `@backstage/techdocs-common`, etc. is on a newer version. Doesn't seem like there's a good reason to pin the version.